### PR TITLE
Removes dev_appserver setting skip_sdk_update_check for now.

### LINKED ISSUE #1044: Cannot start dev_appserver
After creating a project using the new project wizard, the dev_appserver does not start, and gives no output. After stopping the dev_appserver, the following output is dumped:

```
Running dev_appserver.py.
Traceback (most recent call last):
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/dev_appserver.py", line 89, in <module>
    _run_file(__file__, globals())
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/dev_appserver.py", line 85, in _run_file
    execfile(_PATHS.script_file(script_name), globals_)
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 1040, in <module>
    main()
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 1033, in main
    dev_server.start(options)
Allow dev_appserver to check for updates on startup? (Y/n):   File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 771, in start
    update_checker.check_for_updates(configuration)
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/update_checker.py", line 91, in check_for_updates
    if update_check.AllowedToCheckForUpdates():
  File "/usr/local/google/home/alexsloan/tools/google-cloud-sdk/platform/google_appengine/google/appengine/tools/sdk_update_checker.py", line 408, in AllowedToCheckForUpdates
    answer = input_fn('Allow dev_appserver to check for updates on startup? '
EOFError: EOF when reading a line
Terminating dev_appserver.py.
Disconnected from server

```

### PR DESCRIPTION
**Pull request by joaoandremartins**
Fixes #1044

### CODE REVIEW

File: google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form [hunk 1]
```diff
@@ -25,12 +25,14 @@
...
       </component>
-      <grid id="9ed52" binding="appEngineSettingsPanel" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+      <grid id="9ed52" binding="appEngineSettingsPanel" layout-manager="GridLayoutManager" row-count="7" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
         <margin top="0" left="0" bottom="0" right="0"/>
         <constraints>
           <grid row="1" column="0" row-span="10" col-span="2" vsize-policy="3" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
         </constraints>
-        <properties/>
+        <properties>
+          <visible value="true"/>
```
Comment (patflynn) [inline google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form]: what does this change do?

Comment (joaoandremartins) [inline google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form]: I'm removing the "Don't nag..." option from the form.

Another possibility would be to simply hide it, but that would make us consider special cases from skipSdkUpdateCheck being null and I figured we can just roll back this PR once the underlying issue is fixed.

Comment (patflynn) [inline google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form]: isn't that what the delete at line 86 does?

Comment (patflynn) [inline google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form]: @joaoandremartins let's get this in ASAP.

**Comment by joaoandremartins** [comment]
@patflynn I must have misclicked somewhere on the form to add that visible=true. I just fixed it

**Pull request merged**